Editor/RecentSceneToolbarGUI.cs

```diff
@@ -40,9 +40,12 @@ public class RecentSceneToolbarGUI {
 						RecentSceneList.Instance.LoadScene(sceneIndex);
 					});
 			}
-			
-			
+
+#if UNITY_2019_1_OR_NEWER
```

Comment:
"Unable to load the icon" in 2019.1.21 👀 
![image](https://github.com/qwe321qwe321qwe321/Unity-Recent-Scene-Toolbar/assets/2897430/76775c35-763a-4c26-8431-4debdaedd82f)

I think it's been available since 2020.2. https://docs.unity3d.com/2020.2/Documentation/Manual/30_search.html?q=d_ol_plus_act

Reply:
oh, my project is in 2019.4.40f1 and it is available.
I will look into it.

Reply:
OK I tested in 2019.3.0f1. It is available but looks terrible.
![image](https://github.com/qwe321qwe321qwe321/Unity-Recent-Scene-Toolbar/assets/23000374/b3c08f9a-15d9-4d0a-8917-fc7bbcf8f82c)

2019.4
![image](https://github.com/qwe321qwe321qwe321/Unity-Recent-Scene-Toolbar/assets/23000374/dfe2a25a-5ddb-4350-b75f-6292aaad4b5f)

I will change to `UNITY_2019_4_OR_NEWER`.

Reply:
See #2